tests/insurance_test.go

```diff
@@ -90,3 +90,19 @@ func (c *ClientTests) TestInsuranceGetNextPage() {
 		return
 	}
 }
+
+func (c *ClientTests) TestInsuranceRefund() {
+	client := c.TestClient()
+	assert, require := c.Assert(), c.Require()
+
+	insuranceData := c.fixture.BasicInsurance()
+	insuranceData.TrackingCode = "EZ1000000001"
```

Comment:
question: why are we setting `TrackingCode` here? Is there not one in the fixture? If there is, we should remove the redundant line here

Reply:
In test mode, only the `EZ1000000001` tracking status is `pre-transit` which makes the insurance cancellable, the tracking code from the fixture is using a random tracking code.